### DISCUSSION TITLE
What is your egui app?

### DISCUSSION THREAD

**Discussion by mjehrhart** [emilk/egui#1480] [General]
Hello,

I am curious to what you are creating using egui?  If you like, please share a screenshot or a link to your project.

For me, I wrote Project Minty, a rust file deduplication app with egui for the front-end.   Please install and take a look if you like. Any feedback would be great too. 

https://crates.io/crates/minty  

<img width="50%" alt="Screen Shot Minty" src="https://user-images.githubusercontent.com/97703291/163003275-0b4ed1f3-25bd-470a-abf2-77b312fc67d0.gif">

Enjoy,

mje

**Comment by coderedart** [comment]
that is a really great theme. much different from what you normally see with egui. 

just wanted to say that there's already an issue for this kind of stuff where people share their projects and gifs / screenshots.
https://github.com/emilk/egui/issues/996

**Reply by mjehrhart** [reply]
Thanks for the comment! and thanks for pointing me to #996.

**Comment by sameer-coder** [comment]
@mjehrhart Project looks very good!

The fonts look a bit blurry when the light theme is used. I think if egui gets better theming support it will very soon become the most preferred GUI framework.

**Reply by mjehrhart** [reply]
I like egui a lot. A lot of possibilities, wish I was better at UI design.

**Comment by enomado** [comment]
if you want some advices: 

- You dont want such big vertical margins between table rows. 
   (it is contrasting with other vertical margins. And I know the objective you want to achieve by this - row separation. and probably it could work if your other margins would breathe)

- vertical margin from pagination to higher widget should be bigger. ui separation is logic separation. 

- horizontal margins are too way small. they should be proportional to vertical. 

- sliders are too fat. 

Its very IMO.

General rule to perceive the design is to see unfocused, see whole picture, between all widgets. to notice how all starting elements of any widget form one line. Ask yourself why this margin is exact size what it is? what changes if it was bigger/smaller? Use all free space.

**Reply by mjehrhart** [reply]
Thanks for the write-up. I do agree with what you said and very much appreciate it.  My limitation is my knowledge on using this library,  I need to learn a lot more here.

This is the latest version.  This was done after various suggestions from reddit.

![](https://user-images.githubusercontent.com/97703291/164116890-d50354a0-9eef-49e9-b118-d83f0579975c.gif)

**Reply by enomado** [reply]
The most awesome part of experience of this library is that you can click on element and see everything widget is doing, and what styles/margins is using

**Comment by Barugon** [comment]
[Here's](https://github.com/Barugon/cota) my app. Screenshots are in the readme.

I also grabbed a file picker dialog from [dotrix](https://github.com/lowenware/dotrix), made some needed modifications and turned it into a library [here](https://github.com/Barugon/egui_file).
